src/main/java/core/basesyntax/Model/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.Model;
+
+import core.basesyntax.Base.Machine;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
```

Comment:
The message in the `doWork` method should be more informative and indicate that it is an `Excavator` that is working. Refer to the checklist: 'Write informative messages in methods.'